docker-compose.yml

```diff
@@ -128,6 +128,24 @@ services:
                 networks:
                     net:
                         ipv4_address: 10.11.2.2
+        neutrino-switcher:
+                container_name: neutrino-switcher
+                image: lncm/neutrino-switcher:1.0.2
+                logging: *default-logging
+                depends_on: [ bitcoin, lnd ]
+                restart: always
```

Comment:
Also please change the restart policy to `unless-stopped`. All containers should stop when `umbrel-manager` (or anyone else) tries to stop them.

Reply:
Done